contracts/UFragmentsPolicy.sol

```diff
@@ -264,6 +264,40 @@ contract UFragmentsPolicy is Ownable {
         );
     }
 
+    /**
+     * @return Computes the percentage of supply to be added or removed.
+     * Using the function in https://github.com/ampleforth/AIPs/blob/master/AIPs/aip-5.md
+     * @param normalizedRate a DECIMALS decimal fixed point number.
+     */
+    function computeRebasePercentage(
+        int256 normalizedRate, int256 lower, int256 upper, int256 growth)
+        private
+        pure
+        returns (int256)
+    {
+        int256 delta;
+
+        delta = (normalizedRate.sub(ONE));
+
+        // Compute: (Upper-Lower)/(1-(Upper/Lower)/2^(Growth*delta))) + Lower
+
+        int256 exponent = growth.mul(delta).div(ONE);
```

Comment:
@thegostep Is using a fixed point library to handle the `ONES`. You two should sync to see if it makes sense to share it.

Reply:
@thegostep Which library did you end up using, the one I linked or a different one?

Reply:
My current implementation uses https://github.com/HQ20/contracts/tree/master/contracts/math

I like that the operations behave like SafeMath but I have not settled on it.

Here are viable alternatives:
- https://github.com/abdk-consulting/abdk-libraries-solidity
- https://github.com/albertocuestacanada/DecimalMath/blob/master/contracts/DecimalMath.sol
- https://github.com/compound-finance/compound-protocol/blob/master/contracts/Exponential.sol